src/karmen_frontend/src/components/printers/webcam-stream.js

```diff
@@ -10,17 +10,16 @@ const WebcamModal = ({ classNames, source, url, allowFullscreen }) => {
 
   return (
     <>
-      <img
-        className={classNames.join(" ")}
-        alt={`Current state from ${url}`}
-        src={source}
+      <div
+        className={"webcam-stream-image " + classNames.join(" ")}
+        style={{ backgroundImage: `url(${source})` }}
+        title={`Current state from ${url}`}
```

Comment:
This wording isn't helping me understand what it means. Let's replace it with `Last image from ${url}`.

Reply:
Done.